src/Tizen.NUI/src/public/Color.cs

```diff
@@ -207,8 +217,18 @@ public float R
         /// <summary>
         /// The green component.
         /// </summary>
+        /// <remarks>
+        /// The setter is deprecated in API8 and will be removed in API10. Please use new Color(...) constructor.
+        /// </remarks>
+        /// <code>
+        /// // DO NOT use like the followings!
```

Comment:
```suggestion
        /// // DO NOT use as follows:
```

Reply:
thank you for review.
I have fixed by https://github.com/Samsung/TizenFX/pull/4118